packages/components/src/experience-control/style.scss

```
@@ -26,61 +26,68 @@ $experience-types: (
 	)
 );
 
-.experience-control {
+.a8c-experience-control__fieldset {
 	display: flex;
-	flex-direction: column;
-	gap: 12px;
+	gap: 8px;
+	margin: 0;
+    padding: 0;
+    border: none;
```

There's mixed indentation here. This is something that CI should be catching, but it's missing from our checks. This is being fixed in #103396, so you'll want to fix and/or rebase your branch after that's merged to avoid introducing build failures on `trunk`.

```suggestion
	gap: 8px;
	margin: 0;
	padding: 0;
	border: none;
```

---

Thanks for letting me know. This is mostly done by Cursor. Sorry for missing this earlier. Fixed it here: https://github.com/Automattic/wp-calypso/pull/103000/commits/3229a5b6496985e6316b404474d6c2899bf4db16